api/src/pcapi/connectors/serialization/allocine_serializers.py

```diff
@@ -358,6 +359,18 @@ class AllocineShowtime(pydantic.BaseModel):
     def get_first_projection_mode(cls, projection: list[AllocineShowtimeProjection]) -> AllocineShowtimeProjection:
         return projection[0]
 
+    @pydantic.model_validator(mode="before")
+    def validate_languages(cls, data: typing.Any) -> typing.Any:
+        if not isinstance(data, dict):
+            return data
+
+        languages = data.get("languages")
+        if not languages:
+            return data
+
+        data["languages"] = [l for l in languages if l is not None]
+        return data
```

Comment:
Pourquoi ne pas utiliser ici un `field_validator` ? Il me semble que ça serait plus approprié. D'ailleurs je pense qu'un nom plus correct serait `drop_none_values`

```suggestion
    @pydantic. field_validator(mode="before")
    def drop_none_values(cls, langages: list | None) -> list | None:
        if langages:
             return [l for l in languages if l is not None]
        return
```

Reply:
Merci, je ne sais pas pourquoi il me semblait qu'il fallait passer par un `model_validator` (`root_validator` en v1) pour accéder aux données et les filtrer comme ça.